requirements.txt

```diff
@@ -0,0 +1,5 @@
+chardet==3.0.4
+nose==1.3.7
+pycryptodome==3.9.0
+six==1.12.0
```

Comment:
six can be removed after https://github.com/pdfminer/pdfminer.six/pull/305 is merged